### ISSUE TITLE
GetAtt fails

### ISSUE THREAD

**Issue by trondhindenes** [aws-cloudformation/cfn-python-lint#936]
*cfn-lint version: (`cfn-lint --version`)*
`cfn-lint-0.21.3`

*Description of issue.*
this fails (part of template):
```yaml
Outputs: 
  KopsNodeSecurityGroupId: 
    Description: "wakka"
    Value: !GetAtt KopsNodeSecurityGroup.GroupId
```
error:
`yaml.constructor.ConstructorError: could not determine a constructor for the tag '!GetAtt'`

**Comment by trondhindenes** [comment]
this turned out to be another test failing, not cfn-lint. My bad. Closing.
